processor/transformprocessor/internal/metrics/functions_test.go

```diff
@@ -31,6 +31,8 @@ func Test_DataPointFunctions(t *testing.T) {
 
 func Test_MetricFunctions(t *testing.T) {
```

Comment:
I think you'll need to toggle the feature gate in this test.

Reply:
thanks, done